docs/preview/02-Features/02-message-handling/04-event-hubs-azure-functions.md

```diff
@@ -0,0 +1,242 @@
+﻿---
+title: "Azure EventHubs message handling for Azure Functions"
+layout: default
+---
+
+This article describes how you can use Arcus' message handler concept with Azure Functions; allowing you to more easily port your business logic from/to Azure Functions.
+
+# Azure EventHubs message handling for Azure Functions
+While our default message pump system provides a way to receive, route, and handle incoming EventHubs messages which are, unfortunately, not supported in Azure Functions.
+Today, Azure Functions acts as a message receiver meaning that the function is triggered when a message is available but does not handle message routing and handling. However, in this case, it acts as the message pump.
```

Comment:
With the first part of the sentence, are you referring to the 'standard' azure functions implementations ?
And in the second sentence, (the one that starts with 'however', what is 'it' referring to ? .

Reply:
Removed 'however' and restructured it.